mappings/net/minecraft/util/io/PackedIntegerArray.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3508 net/minecraft/util/PackedIntegerArray
+CLASS net/minecraft/class_3508 net/minecraft/util/io/PackedIntegerArray
```

Comment:
This is not IO but like some other chunk stuff?

Reply:
It's related to storage. Storage is a subset of IO.

Reply:
This sounds like collection as it's handling arrays.

Reply:
Moved. However this class _does_ seem to refer to itself as "storage" so perhaps a rename might be in order:

`throw Util.method_22320(new RuntimeException("Invalid length given for storage, got: " + ls.length + " but expected: " + k));`